src/components/activity.vue

```diff
@@ -182,7 +206,16 @@ ion-card-header {
   display: flex;
   align-items: center;
 }
+.no-padding{
+  padding: 0px 13px;
+}
+.marginRgt{
+  margin-right: 10px;
+}
```

Comment:
```suggestion
```

please use the ionic-padding classes rather than building new ones: https://ionicframework.com/docs/layout/css-utilities#element-padding